src/service/blogService.js

```
@@ -32,7 +40,29 @@ class BlogService {
+  }
+
+  static async uploadImage(imageData) {
+    try {
+      const response = await axiosInstance.post(`/upload`, imageData, {
+        headers: {
+          "Content-Type": "multipart/form-data",
+        },
+      });
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)